packages/front-end/components/Experiment/EditVariationsForm.tsx

```diff
@@ -111,14 +114,17 @@ const EditVariationsForm: FC<{
         setVariations={(v) => {
           form.setValue(
             "variations",
-            v.map((data, i) => {
+            v.map((data) => {
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
nit: We can do this to make TypeScript happy

```
const { value, ...newData } = data;
return {
  // default values
  name: "",
  description: "",
  screenshots: [],
  ...newData,
  key: value,
};
```

Reply:
good suggestion, thanks. fixed